contributors/guide/non-code-contributions.md

```diff
@@ -18,14 +18,15 @@ Are you interested in any of the roles below? Come chat with us [on Slack](https
 These are roles that either span the project as a whole, or span several areas of the project. Most of the roles below can be considered "good-first-roles".
 
 - Community education
-  - Answering questions on Discuss, Slack, StackOverflow, etc
-  - K8s Office Hours
-  - Meet Our Contributors
+  - Answering questions on [Discuss](https://discuss.kubernetes.io/), [Slack](http://slack.k8s.io/), [StackOverflow](https://stackoverflow.com/questions/tagged/kubernetes), etc
+  - K8s Office Hours - [Slack](https://kubernetes.slack.com/messages/C6RFQ3T5H/), [Discuss](https://discuss.kubernetes.io/tags/office-hours), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg)
+  - Meet Our Contributors - [Slack](https://kubernetes.slack.com/messages/C8WRR2BB9/), [YouTube](https://www.youtube.com/playlist?list=PL69nYSiGNLP3QpQrhZq_sLYo77BVKv09F)
```

Comment:
Can you add in https://github.com/kubernetes/community/blob/master/mentoring/meet-our-contributors.md here?

Reply:
done